app/src/main/java/org/simple/clinic/facility/change/FacilityChangeActivity.kt

```diff
@@ -158,15 +156,6 @@ class FacilityChangeActivity : AppCompatActivity() {
   }
 
   fun updateFacilities(facilityItems: List<FacilityListItem>, updateType: FacilitiesUpdateType) {
-    // Avoid animating the items on their first entry.
```

Comment:
Any particular reason for removing this?

Reply:
Because we are using `ItemAdapter` which will ship with default animations already. With the current approach, we were adding additional complexity to track whether the animations were already shown once or not for little benefit.